src/utils/renderHtmlDocument.ts

```diff
@@ -72,8 +72,8 @@ export const renderHtmlDocument = ({ html, css }: TemplateData) =>
             }
             @font-face {
                 font-family: "GuardianTextEgyptian";
-                src: url(${CDNFontPath}/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-RegularBold.woff2) format("woff2"),
-                    url(${CDNFontPath}/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-RegularBold.woff) format("woff");
+                src: url(${CDNFontPath}/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Bold.woff2) format("woff2"),
```

Comment:
🎉

Reply:
Definitely not a blocker/for this PR, but I wonder if we should require the platform to provide the fonts. As it's easy to imagine most components will require them.

Reply:
As discussed offline, we're only including the fonts in our preview template, not sending them as part of the component's styling. We do this on the assumption that our platforms will _all_ make use of those fonts and have their own ways of making them available in the document.